modules/checking-cluster-resources-after-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,223 @@
++
+[source,terminal]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: <example_egress_policy>
+  namespace: <namespace>
+spec:
+  egress:
+  - type: Allow
+    to:
+      cidrSelector: 192.168.0.0/16
+  - type: Deny
+    to:
+      cidrSelector: 0.0.0.0/0
+----
++
+Ensure that the behavior of this resource is intended because it could have changed after migration. For more information about egress firewalls, see "Configuring an egress firewall for a project".
+
+.. To check for egress firewall resources by using the {product-title} web console:
+
+... On the {product-title} web console, click *Observe* -> *Metrics*.
+
+... In the *Expression* box, type `ovnkube_controller_num_egress_firewall_rules` and click *Run queries*. If you have egress firewall resources, they are returned in the *Expression* box.
+
+. Check your cluster for egress IP resources. You can do this by using the `oc` CLI, or by using the {product-title} web console.
+
+.. To check for egress IPs by using the `oc` CLI tool:
+
+... To list the namespace with egress IP resources, enter the following command:
++
+[source,terminal]
+----
+$ oc get egressip
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                EGRESSIPS    ASSIGNED NODE                              ASSIGNED EGRESSIPS
+egress-sample       192.0.2.10   ip-10-0-42-79.us-east-2.compute.internal   192.0.2.10
+egressip-sample-2   192.0.2.14   ip-10-0-42-79.us-east-2.compute.internal   192.0.2.14
+----
+
+... To provide detailed information about an egress IP, enter the following command:
++
+[source,terminal]
+----
+$ oc get egressip <egressip_name> -o yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressIP
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"k8s.ovn.org/v1","kind":"EgressIP","metadata":{"annotations":{},"name":"egressip-sample"},"spec":{"egressIPs":["192.0.2.12","192.0.2.13"],"namespaceSelector":{"matchLabels":{"name":"my-namespace"}}}}
+  creationTimestamp: "2024-06-27T15:48:36Z"
+  generation: 7
+  name: egressip-sample
+  resourceVersion: "125511578"
+  uid: b65833c8-781f-4cc9-bc96-d970259a7631
+spec:
+  egressIPs:
+  - 192.0.2.12
+  - 192.0.2.13
+  namespaceSelector:
+    matchLabels:
+      name: my-namespace
+----
++
+Repeat this for all egress IPs. Ensure that the behavior of each resource is intended because it could have changed after migration. For more information about EgressIPs, see "Configuring an EgressIP address".
+
+.. To check for egress IPs by using the {product-title} web console:
+
+... On the {product-title} web console, click *Observe* -> *Metrics*.
+
+... In the *Expression* box, type `ovnkube_clustermanager_num_egress_ips` and click *Run queries*. If you have egress firewall resources, they are returned in the *Expression* box.
+
+. Check your cluster for multicast enabled namespaces. You can only do this by using the `oc` CLI.
```

Comment:
```suggestion
. Check your cluster for multicast enabled namespaces. You can only do this by using the `oc` CLI:
```
I won't keep harping on the "colon leading into substeps", but check for those throughout.